src/PKSim.Infrastructure/ORM/Repositories/OntogenyRepository.cs

```diff
@@ -115,6 +184,29 @@ private Ontogeny forMolecule(OriginData originData, string molecule)
          return allFor(originData.Species.Name).FindByName(molecule) ?? new NullOntogeny();
       }
 
+      public DataRepository OntogenyToRepository(Ontogeny ontogeny, string containerName)
+      {
+         var dataRepository = new DataRepository {Name = PKSimConstants.UI.OntogenyFor(ontogeny.DisplayName, containerName)};
+         var xUnit = _displayUnitRetriever.PreferredUnitFor(_dimensionRepository.AgeInWeeks);
+         var yUnit = _displayUnitRetriever.PreferredUnitFor(_dimensionRepository.Fraction);
```

Comment:
I think it should be `dimensionless `and not `fraction `here?

Reply:
Fixed it in another PR too